### PR TITLE
Hw 6 branch

### PR DESCRIPTION
**Pull request by azartnyi**
Hello, check please <3

### CODE REVIEW

File: HW_5/Task_2.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+let bytes = 1000000000000;
+
+const KB = 1024;
+const MB = KB * 1024;
+const GB = MB * 1024;
+const TB = GB * 1024;
+
+let result;
+
+if (bytes >= TB) {
```
Comment (anatoly-karpovich) [inline HW_5/Task_2.js:12]: better do this with while loop

File: HW_5/Task_3.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const word = "example";
+let vowels = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'];
+let vowelCount = 0;
+let consonantCount = 0;
+
+for (let i = 0; i < word.length; i++) {
+    if (vowels.includes(word[i])) {
+        vowelCount++;
+    } else {
+        consonantCount++;
```
Comment (anatoly-karpovich) [inline HW_5/Task_3.js:14]: word can contain "-" symbol so need to check for consonantes separately

File: HW_6/Task_1.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    const index = employeeNames.indexOf(employeeName);
+
+    if (index === -1) {
+        return null;
+    }
+
+    const nameAndSalary = [];
+    nameAndSalary.push(employeeNames[index]);
+    nameAndSalary.push(employeeSalary[index]);
+    return nameAndSalary;
```
Comment (anatoly-karpovich) [inline HW_6/Task_1.js:23]:
```suggestion
    return [employeeNames[index], employeeSalary[index]];
```

File: HW_6/Task_2.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
+// // 1. У вас есть массив названий пицц вашего конкурента. Создайте функцию, которая будет принимать ваш набор названий пицц (массив) 
+//   и возвращать только те, которых нет у конкурента (тоже массив). Если все ваши пиццы есть у конкурента - вернуть null
+//   Пиццы конкурента:
+//   const competitorPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai']
+
+const competitorPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai'];
+
+function getUniquePizzas(myPizzas, competitorPizzas) {
+  const uniquePizzas = myPizzas.filter(pizza => !competitorPizzas.includes(pizza));
```
Comment (anatoly-karpovich) [inline HW_6/Task_2.js:9]: need to convert to lowerCase first. Function needs to be case insencitive

**Review by anatoly-karpovich — APPROVED** [review]

**Review by VladimirLQA — APPROVED** [review]

**Pull request merged**